flink-table/flink-sql-parser-hive/src/main/codegen/data/Parser.tdd

```diff
@@ -128,6 +129,7 @@
     "IF"
     "INPUTFORMAT"
     "ITEMS"
+    "JAR"
```

Comment:
It is possible to add this into non-reserved keywords? Otherwise, users use `jar` as field names will not work when upgrading to new version.